src/instructions/create_instructions.py

```
@@ -3,6 +3,37 @@
+def get_row_stitch_sequences(connectivity, creases) -> list[list[str]]:
+    """Convert per-row DTW connectivity into per-row stitch instruction sequences.
+
+    Args:
+        connectivity: sequence of per-row-pair DTW paths
+        creases: sequence of per-row crease labels
+
+    Returns:
+        list[list[str]]: one stitch-instruction list per row; every other row is reversed (back-and-forth crochet)
+            to avoid shape deviation from the slant of the stitches.
+    """
+    return [dtw_to_stitches(dtw, bflo)[::(-1) ** i] for i, (dtw, bflo) in enumerate(zip(connectivity, creases))]
```

Ruff (0.15.20)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)